api/fixtures/example.go

```diff
@@ -34,7 +34,6 @@ func (o *ExampleResources) AsObjects() []crclient.Object {
 	objects := []crclient.Object{
 		o.Namespace,
 		o.PullSecret,
-		o.Cluster,
 	}
 	objects = append(objects, o.Resources...)
```

Comment:
worth to mention that the required infra-credentials secret is one of the objects at `o.Resources`